packages/avatars/src/styled/StyledStandaloneStatusCaption.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'avatars.status-indicator.caption';
+
+function sizeStyles(props: ThemeProps<DefaultTheme>) {
+  return css`
+    padding: 0 ${props.theme.space.base}px;
+    line-height: ${props.theme.space.base * 6}px;
+    font-size: ${props.theme.fontSizes.md};
+    font-weight: ${props.theme.fontWeights.regular};
```

Comment:
It shouldn't be necessary to specify `font-weight` here; accepting the cascade would be desirable in most cases.